src/main/java/seedu/internbuddy/model/Model.java

```diff
@@ -8,7 +8,7 @@
 import seedu.internbuddy.model.company.Company;
 
 /**
- * The API of the Model component.
+ * The API of the ModelCompany component.
```

Comment:
Think might be appropriate to leave it as Model component for accuracy to the filename